index.html

```diff
@@ -16,19 +16,22 @@
     <nav class="main-nav">
       <div id="hamburger"><i class="icon-menu"></i></div>
       <ul>
-        <li>Bio</li>
-        <li>Projects</li>
-        <li>Contact</li>
+        <li><a href="#" class="icon-home tab" data-content="bioInfo">Bio</a></li>
```

Comment:
The data content logic you applied to Bio and Projects can be applied to contact as well. That way contact is hidden until the user clicks on contact!